app-containers/aardvark-dns/aardvark-dns-1.8.0.ebuild

```diff
@@ -154,11 +13,13 @@ if [[ ${PV} == *9999* ]]; then
 	EGIT_REPO_URI="https://github.com/containers/aardvark-dns.git"
 else
 	SRC_URI="${CARGO_CRATE_URIS}"
+	SRC_URI+="https://github.com/containers/aardvark-dns/releases/download/v${PV}/${PN}-v${PV}-vendor.tar.gz"
 	KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv"
 fi
 LICENSE="0BSD Apache-2.0 Apache-2.0-with-LLVM-exceptions MIT Unlicense Unicode-DFS-2016 ZLIB"
 SLOT="0"
-QA_FLAGS_IGNORED="usr/bin/${PN}"
```

Comment:
If portage is configured for this warning then it will trigger it unless we set QA_FLAGS_IGNORED (see [bug 796887](https://bugs.gentoo.org/796887)):
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/libexec/podman/aardvark-dns
```

Reply:
Added in effa8d7a3c8e69c87bb9c765f73d9f557fa20f01.

Reply:
How to configure portage to see this warning? I didnt see this one. I saw prestrip one.

Reply:
Try this: https://gist.github.com/thesamesam/1a48cee5cc9584dd70604be22e544557